.aliases

```diff
@@ -20,6 +20,12 @@ alias dot='pushd ~/dotfiles'
 alias fig='vim $(fzf --height=40%)'
   alias vm='vim'
 
+# alias to fix cross-platform issue
+alias aws.exe='aws'
```

Comment:
This doesn't actually solve the issue & it could theoretically cause issue down the line. Should be removed.